clients/linux/src/backend.rs

```diff
@@ -342,16 +361,69 @@ impl LbCore {
         Ok((meta, String::from_utf8_lossy(&decrypted).to_string()))
     }
 
-    pub fn sync_status(&self) -> LbResult<String> {
+    pub fn sync_status(&self, just_synced: bool) -> LbResult<String> {
         match self.get_last_synced()? {
             0 => Ok("✘  Never synced.".to_string()),
             _ => {
-                let work = self.calculate_work()?;
+                let work = match calculate_work(&self.config) {
+                    Ok(ok) => ok,
+                    Err(err) => {
+                        if let UiError(CalculateWorkError::CouldNotReachServer) = err {
+                            return Ok("You are offline".to_string());
+                        }
+
+                        return Err(match_core_err!(err, CalculateWorkError,
+                            CouldNotReachServer => uerr!("Unable to connect to the server."),
+                            ClientUpdateRequired => uerr!("Client upgrade required."),
+                            NoAccount => uerr!("No account found."),
+                            @Unexpected(msg) => return Err(progerr!("{}", msg)),
+                        ));
+                    }
+                };
+
                 let n_files = work.work_units.len();
+
+                if just_synced {
+                    return Ok(match n_files {
+                        0 => "✔  Synced.".to_string(),
+                        _ => format!(
+                            "<b>{}</b>  file{} not synced.",
+                            n_files,
+                            if n_files == 1 { "" } else { "s" }
+                        ),
+                    });
+                }
+
+                let usage = self.local_and_server_usage()?;
```

Comment:
I don't think this stuff should end up here because it's not related to the "sync status". I think this may be the trickiest bit of this PR. Here are the tricky bits:

+ At what point is it appropriate to call this?
+ How do we handle competing information?
+ What portions of core do we have to modify to make this a better experience for a linux developer?

Here are some considerations:
+ set_sync_status is called every time a file is edited, this means that we're recomputing usage every time, which involves a round trip to the server. This current version is also [calculating my compression ratio](https://github.com/lockbook/lockbook/blob/master/core/src/service/usage_service.rs#L59), which is *really* expensive.
+ To the same UI element we're going to be sending competing information. Sometimes we'll send that a file is synced, or is dirty or that the user is approaching their cap, etc. There is an ordering of importance (as described in that doc), what's the best way to hang on to these competing pieces of data and show the most relevant one to the user.
+ So you'll likely only want to calculate usage when a sync is finished, you'll likely not want to calculate the compression ratio, you'll definitely not want to parse strings.

When set_sync_status and set_usage_status become two separate entities (because they're called at different times) the logic about what info is shown at the current moment moves somewhere else. Probably somewhere closer to the UI, and as this is a UI decision, it's probably a good one.

This may get pushed out into a separate PR if you find it too tangential, but: the final thing I want you to think about while you sort this out: I don't want to open the app and for a moment see "Synced!" and then see "Syncing...." and then see upto date. If the init step is to kick off a sync, then initially I should see "Syncing...". Ideally the user never sees something for a split second and is left wondering what that said. In the app the only place this sort of complexity and fine tuning is needed is this sync_status / bottom bar region. By fine tuning I mean, if we show a sync progress bar, maybe we delay the sync operation starting by 0.25s and we delay it's completion by 0.25 seconds so we ensure that "Syncing..." stays on screen for at least half a second. This is probably something that should happen everywhere (@tvanderstad)